client/src/services/translation.ts

```diff
@@ -53,6 +54,46 @@ export function getWorkspaceRoleTranslationKey(role: WorkspaceRole | null): Work
   }
 }
 
+interface CustomOrderStatusTranslations {
+  title: Translatable;
+  description?: Translatable;
+}
+
+export function getCustomOrderStatusTranslationKey(status: CustomOrderStatus | undefined): CustomOrderStatusTranslations {
```

Comment:
`undefined` should be needed here, `CustomOrderStatus` has an `Unknown` value that can be used.